testutil/keeper/bank.go

```diff
@@ -13,3 +13,9 @@ func (tk TestKeepers) Mint(ctx sdk.Context, address string, coins sdk.Coins) {
 	require.NoError(tk.T, tk.BankKeeper.MintCoins(ctx, minttypes.ModuleName, coins))
 	require.NoError(tk.T, tk.BankKeeper.SendCoinsFromModuleToAccount(ctx, minttypes.ModuleName, sdkAddr, coins))
 }
+
+// MintModule mints the specified coins into the module account balance
+func (tk TestKeepers) MintModule(ctx sdk.Context, moduleAcc string, coins sdk.Coins) {
+	require.NoError(tk.T, tk.BankKeeper.MintCoins(ctx, minttypes.ModuleName, coins))
+	require.NoError(tk.T, tk.BankKeeper.SendCoinsFromModuleToModule(ctx, minttypes.ModuleName, moduleAcc, coins))
```

Comment:
```suggestion
	require.NoError(tk.T, tk.BankKeeper.MintCoins(ctx, moduleAcc, coins))
```

Reply:
What if the module does not have mint permissions?

Reply:
I think it is cleaner (and fail-safe) this way, tbh

Reply:
Yes you're right